tests/test_constants.py

```diff
@@ -0,0 +1,10 @@
+from sympl import set_constant
+import pytest
+
+
+def test_setting_wrong_units():
+
+    with pytest.raises(TypeError) as excinfo:
+        set_constant('abcd', 100, 'Wii')
+
+    assert 'valid unit' in str(excinfo.value)
```

Comment:
Should add a few more tests here, since adding this one test makes the code "covered" and makes it hard to realize we need more tests. Particularly these three unit tests:

- test setting a valid unit
- test that changing the value for an already-existing quantity actually changes both the value and the units
- test that setting a new value that doesn't already exist defines a new constant in default_constants with the correct value and units

Reply:
added.